src/lib.rs

```diff
@@ -344,6 +344,8 @@ pub use self::ser::{
 };
 #[doc(inline)]
 pub use self::value::{from_value, to_value, Map, Number, Value};
+#[cfg(feature = "raw_value")]
+pub use self::value::to_raw;
```

Comment:
Please remove this reexport. This is fine staying in the `value` module with RawValue.